test/Microsoft.Azure.SignalR.AspNet.Tests/E2ETest/TestClientSet.cs

```diff
@@ -0,0 +1,70 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNet.SignalR.Client;
+using Microsoft.Azure.SignalR.Tests.Common;
+
+namespace Microsoft.Azure.SignalR.AspNet.Tests
+{
+    public class TestClientSet : ITestClientSet
+    {
+        private IList<HubConnection> _connections;
+        private IList<IHubProxy> _proxies;
+
+        public int Count {
+            get
+            {
+                if (_connections == null)
+                {
+                    throw new ArgumentNullException(nameof(_connections));
+                }
+                return _connections.Count;
+            }
+        }
+
+        public void AddListener(string methodName, Action<string> handler)
+        {
+            foreach (var p in _proxies)
+            {
+                p.On(methodName, handler);
+            }
+        }
+
+        public Task AllSendAsync(string methodName, string message)
+        {
+            return Task.WhenAll(from p in _proxies
+                                select p.Invoke(methodName, message));
+        }
+
+        public ITestClientSet Create(string serverUrl, int count)
```

Comment:
>Create [](start = 30, length = 6)

why not in ctor?

Reply:
It is an interface, ctor seems not able to be  an interface.